sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/models/TranslationText.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.translation.text.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Translation result.
+ */
+@Immutable
+public final class TranslationText {
+
+    /*
+     * A string representing the language code of the target language.
+     */
+    @Generated
+    @JsonProperty(value = "to")
+    private final String to;
+
+    /*
+     * A string giving the translated text.
+     */
+    @Generated
+    @JsonProperty(value = "text")
+    private final String text;
+
+    /*
+     * An object giving the translated text in the script specified by the toScript parameter.
+     */
+    @Generated
+    @JsonProperty(value = "transliteration")
+    private TransliteratedText transliteration;
+
+    /*
+     * Alignment information.
+     */
+    @Generated
+    @JsonProperty(value = "alignment")
+    private TranslatedTextAlignment alignment;
+
+    /*
+     * Sentence boundaries in the input and output texts.
+     */
+    @Generated
+    @JsonProperty(value = "sentLen")
+    private SentenceBoundaries sentenceBoundaries;
+
+    /**
+     * Creates an instance of TranslationText class.
+     *
+     * @param to the to value to set.
+     * @param text the text value to set.
+     */
+    @Generated
+    @JsonCreator
+    private TranslationText(@JsonProperty(value = "to") String to, @JsonProperty(value = "text") String text) {
+        this.to = to;
+        this.text = text;
+    }
+
+    /**
+     * Get the to property: A string representing the language code of the target language.
+     *
+     * @return the to value.
+     */
+    @Generated
+    public String getTo() {
```

Comment:
We should change this to `getTargetLanguage()` to match the names used in the client methods.

Reply:
Updated